### ISSUE TITLE
Cannot access type interfaces from the `browser` API as `@types/chrome`.

### ISSUE THREAD

**Issue by 1natsu172** [wxt-dev/wxt#1213]
The second thing is that the interface _as a type_ cannot be inferred from the `browser` as you say. This one is not a package-manager dependent problem. I think it's an implementation glitch in WXT.

The workaround over here is below, and either approach should be used.

```typescript
import { Runtime } from "wxt/browser";

// This types from webextension-polyfill
function handler1(params: Runtime.MessageSender) {}

// This types from @types/chrome(need install `@types/chrome`)
function handler2(params: chrome.runtime.MessageSender) {}
```

_Originally posted by @1natsu172 in https://github.com/wxt-dev/wxt/issues/1116#issuecomment-2440652997_

and

_Originally posted in https://github.com/wxt-dev/wxt/issues/1116#issuecomment-2440320040_

**Comment by 1natsu172** [comment]
After some investigation, it seems this might be more of an issue with the upstream type definitions rather than WXT itself. Since the interfaces are defined alongside the implementation within a namespace, there is no way to re-export them.

**Comment by aklinker1** [comment]
Yeah, is this just how `@types/chrome` works. I have a section on using its types in the docs here:

https://wxt.dev/guide/essentials/extension-apis.html#disabling-the-polyfill
